src/Contract/Transaction.purs

```diff
@@ -135,37 +105,77 @@ import Cardano.Types.Transaction -- Most re-exported, don't re-export `Redeemer`
   , _withdrawals
   , _witnessSet
   ) as Transaction
+import Cardano.Types.Transaction (Transaction, _body, _inputs)
+import Contract.Monad (Contract, liftedE, liftedM, wrapContract)
+import Control.Monad.Error.Class (try, catchError, throwError)
+import Control.Monad.Except.Trans (runExceptT)
+import Control.Monad.Reader (asks, runReaderT, ReaderT)
+import Data.Array as Array
+import Data.Array.NonEmpty as NonEmptyArray
+import Data.Either (Either(..), hush)
```

Comment:
Explicit imports

Reply:
Even when importing qualified? I didn't realize that.